observability/logging/logging-6.1/log6x-clf-6.1.adoc

```diff
@@ -62,9 +62,9 @@ Filters:: Transform or drop log messages in the pipeline. Users can define filte
 
 Inputs are configured in an array under `spec.inputs`. There are three built-in input types:
 
-application:: Selects logs from all application containers, excluding those in infrastructure namespaces such as `default`, `openshift`, or any namespace with the `kube-` or `openshift-` prefix.
+application:: Selects logs from all application containers, excluding those in infrastructure namespaces such as `default`, `kube`, `openshift`, or any namespace with the `kube-` or `openshift-` prefix.
```

Comment:
Similar to 6.0 review...

Would it be better to exclude the ending 
> such as default, kube, openshift, or any namespace with the kube- or openshift- prefix.

and just define it in the infrastructure definition below?

Reply:
done